src/views/home/components/header/header.styles.ts

```diff
@@ -0,0 +1,64 @@
+import { createUseStyles } from "react-jss";
+
+import { Theme } from "src/styles/theme";
+
+const flexBox = { flex: 1, display: "flex" };
+const btn = (theme: Theme) => ({
+  display: "inline-flex",
+  alignItems: "center",
+  padding: [[theme.spacing(0.75), theme.spacing(1)]],
+  borderRadius: theme.spacing(1),
+  height: theme.spacing(4.5),
+  "& svg": {
+    width: 24,
+  },
+  "& p": {
+    margin: [[0, theme.spacing(1)]],
+    display: "none",
+    [theme.breakpoints.upSm]: {
+      display: "inherit",
+    },
+  },
+  "&:hover": {
+    backgroundColor: theme.palette.grey.light,
+  },
+});
+
+const useHeaderHomeStyles = createUseStyles((theme: Theme) => ({
+  header: {
+    ...flexBox,
+    fontWeight: theme.fontWeights.medium,
+    "& > *": {
+      flex: 1,
+    },
+  },
+  settingsBtn: {
+    ...flexBox,
+    alignItems: "center",
+    "& a": {
+      ...btn(theme),
+    },
+  },
+  myAccountBtn: {
+    ...flexBox,
+    alignItems: "center",
+    justifyContent: "right",
+    "& a": {
+      ...btn(theme),
+    },
+  },
+  logo: {
+    alignItems: "center",
+    textAlign: "center",
+    flexDirection: "column",
+    "& svg": {
+      width: 43,
+    },
+    "& p": {
```

Comment:
Avoid targeting inner elements, what if we change from a `p` to an `h1`? It's always better to have an specific class for each element, like `logoText`. Far more robust to changes.